demo/select-stt-json-type.js

```diff
@@ -18,6 +18,7 @@ const SttTypeSelect = props => {
     <option value="vtt-youtube" disabled>Youtube VTT</option>
     <option value="amazontranscribe">Amazon Transcribe</option>
     <option value="digitalpaperedit">Digital Paper Edit</option>
+    <option value="gcp">Google STT</option>
```

Comment:
It be good to rename `gcp` to something more consistent with the other adapters eg `google-stt` `google-cloud-stt`, `google-cloud-platform` etc..

```suggestion
    <option value="google-stt">Google STT</option>
```